modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -260,7 +258,7 @@ object SeqexecEngine {
                       odbQueuePollingInterval: Duration)
   val defaultSettings = Settings(Site.GS,
     "localhost",
-    LocalDate.of(2017, 1,1),
+    LocalDate.of(2017, 1, 1),
```

Comment:
One of the attributes of `Settings` is a date, therefore `defaultSettings` needs a date. `defaultSettings` is used on the tests as a base instance of Settings, from which a new one is built with specific values for some attributes.